src/RemoteLRS.php

```diff
@@ -924,6 +910,7 @@ public function deleteActivityProfile($activity, $id) {
                     'activityId' => $activity->getId(),
                     'profileId'  => $id,
                 ),
+                'ignore404' => true
```

Comment:
Is this from a spec change? Retrieving makes sense to ignore, but the user may want to know if they tried to delete something that they thought existed.